Modix.Services/Configuration/DiscordBotConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace Modix.Services.Configuration
+{
+    public class DiscordBotConfiguration
```

Comment:
We should consider adding these configuration properties to the existing [`ModixConfig`](https://github.com/discord-csharp/MODiX/blob/ec29a4557fd38c2dc3d7ecfde69221ca4fd3df7d/Modix.Data/Models/ModixConfig.cs) class that's elsewhere in the project.

Reply:
I kind of wanted to keep them separate, seeing as one is for environmental purposes and one is for behavior

Reply:
I don't know.. I think there'd need to be a _really_ good reason to have two separate configuration classes (especially with a name as generic as this). Maybe the database is a better place for toggling this if it's meant to be per-instance rather than per-environment?